rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -612,7 +610,8 @@ TEST_F(RosBag2PlayQosOverrideTestFixture, override_has_precedence_over_recorded)
   const rclcpp::Duration override_liveliness_offer{250ms};
   ASSERT_LT(liveliness_request, recorded_liveliness_offer);
   ASSERT_LT(override_liveliness_offer, liveliness_request);
-  const auto request_profile = Rosbag2QoS{}.liveliness_lease_duration(liveliness_request);
+  const auto request_profile = Rosbag2QoS{}.liveliness_lease_duration(
+    liveliness_request);
```

Comment:
```suggestion
  const auto request_profile = Rosbag2QoS{}.liveliness_lease_duration(liveliness_request);
```
Nitpick. Could be in one line.